features/io.asgardio.java.saml.sdk.server.feature/pom.xml

```diff
@@ -19,23 +19,23 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
     <parent>
-        <groupId>org.wso2.carbon.identity.agent.sso.java</groupId>
-        <artifactId>identity-agent-sso</artifactId>
+        <groupId>io.asgardio.java.saml.sdk</groupId>
+        <artifactId>java-saml-sdk</artifactId>
         <relativePath>../../pom.xml</relativePath>
         <version>5.5.9-SNAPSHOT</version>
```

Comment:
Update versions in a separate commit

Reply:
Resolved in https://github.com/gayashanbc/identity-agent-sso/pull/2/commits/0785a407e47099951350f5e95fecd0f16e4d6c13